rosidl_runtime_rs/src/string.rs

```diff
@@ -213,6 +213,11 @@ macro_rules! string_impl {
             }
         }
 
+        // SAFETY: A string is a simple data structure, and therefore not thread-specific.
+        unsafe impl Send for $string {}
```

Comment:
Forgive the potentially silly question, but I am not familiar with the `$string` syntax. What does it mean?

Reply:
This is inside a macro. `$string` can be either the `String` or `WString` type.